general-superstaq/general_superstaq/superstaq_client.py

```diff
@@ -111,17 +114,29 @@ def __init__(
         }
         self.session = requests.Session()
 
-        if ibmq_channel and ibmq_channel not in ("ibm_quantum", "ibm_cloud"):
-            raise ValueError("ibmq_channel must be either 'ibm_cloud' or 'ibm_quantum'.")
-
         if cq_token:
             kwargs["cq_token"] = cq_token
-        if ibmq_token:
-            kwargs["ibmq_token"] = ibmq_token
-        if ibmq_instance:
-            kwargs["ibmq_instance"] = ibmq_instance
-        if ibmq_channel:
-            kwargs["ibmq_channel"] = ibmq_channel
+
+        if use_stored_ibmq_credentials:
+            ibmq_provider = IBMProvider()
```

Comment:
for runtime i think this will be something like
```suggestion
            ibmq_account = qiskit_ibm_runtime.accounts.AccountManager.get(
                channel=ibmq_channel, name=ibmq_name
            )
```
qiskit runtime allows multiple accounts to be saved to be saved under different names, so we might want to add an argument to allow users to specify an `ibmq_name` too?